src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -282,7 +282,7 @@ public async Task<IActionResult> ExternalLoginCallback(string returnUrl = null,
         {
             if (remoteError != null)
             {
-                _logger.LogError("Error from external provider: {Error}", remoteError);
+                _logger.LogError($"Error from external provider: {remoteError}");
```

Comment:
Localize the log message

Reply:
Well this change is wrong, revert it.

Reply:
I merged dev branch, keep target accountcontroller and manually did the changes. But when I commited the merge, the AccountController was not staged and left uncommited.
That happens when instead of sleeping you merge :)